### PR TITLE
Wizard Component  Refactors

### PR DESCRIPTION
**Pull request by Syeda-Azim**
Setting Alloy developer as the only Owner of this Repo and fixing dependency issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "@babel/polyfill": "^7",
     "@brightspace-ui/stylelint-config": "^0.0.3",
     "@webcomponents/webcomponentsjs": "^2",
-    "babel-eslint": "^10",
+    "@babel/eslint-parser": "^7",
```
Comment (Syeda-Azim) [inline package.json:46]: Error:
image
Fix:
https://d2l.slack.com/archives/C0PHG3QB0/p1614090224003900

**Comment by IhabAlkabary** [comment]
Looks good. But can you re-try CI since it's stuck in Test

**Comment by Syeda-Azim** [comment]
> Looks good. But can you re-try CI since it's stuck in Test

Unit test is in the **CI /Test(pull_request)** job. There is no setup to run any sepearate **test** job.
https://github.com/BrightspaceUILabs/wizard/pull/38/checks?check_run_id=2642909290

**Comment by IhabAlkabary** [comment]
> > Looks good. But can you re-try CI since it's stuck in Test
> 
> Unit test is in the **CI /Test(pull_request)** job. There is no setup to run any sepearate **test** job.
> https://github.com/BrightspaceUILabs/wizard/pull/38/checks?check_run_id=2642909290

Then if we can get rid of that step, doesn't have to be part of this PR, that would be great.

**Comment by IhabAlkabary** [comment]
Shall we bump the minor version?

**Comment by IhabAlkabary** [comment]
> Shall we bump the minor/patch version?

I see you have fix: in he commit, so it will be a patch!

**Review by IhabAlkabary — APPROVED** [review]

**Pull request merged**